nuxeo-web-ui-ftest/webdriver/test/features/step_definitions/support/fixtures/users.js

```diff
@@ -40,10 +40,14 @@ module.exports = function () {
       .map((user) => {
         if (user !== 'Administrator') {
           const userWorkspace = `/default-domain/UserWorkspaces/${user}`;
-          nuxeo.repository().delete(userWorkspace).catch(() => {});
-          fixtures.users.delete(user);
+          const userFavorites = `${userWorkspace}/Favorites`;
+          return nuxeo.repository().delete(userFavorites)
+                         .catch(() => {}) // eslint-disable-line arrow-body-style
+                         .then(() => nuxeo.repository().delete(userWorkspace))
```

Comment:
Since we're already deleting the user workspace, we don't have to delete something inside it on another cleanup routine (see https://github.com/nuxeo/plugin-nuxeo-web-ui/blob/master/nuxeo-web-ui-ftest/webdriver/test/features/step_definitions/support/fixtures/collections.js#L19). Doing so will throw a 404.